modules/examples/todolist/client/src/main/scala/ClientApp.scala

```diff
@@ -31,7 +31,8 @@ object ClientApp {
 
     logger.info(s"${Thread.currentThread().getName} Starting client...")
 
-    clientProgram[IO].unsafeRunSync()
+    pongProgram[IO].unsafeRunSync()
+    tagProgram[IO].unsafeRunSync()
```

Comment:
You could run both programs with just one effect, for example:

```scala
import cats.syntax.apply._
(pongProgram[IO] *> tagProgram[IO]).unsafeRunSync()
```

Reply:
Thanks!